src/app/docs/input-box/input-box-docs.component.html

```diff
@@ -0,0 +1,222 @@
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Hover
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-hover.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Focus
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-focus.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Click
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-click.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Disabled
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-disabled.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Required
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-required.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Error
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-error.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Make the input label succinct and descriptive. Don't construct sentences or ask questions with the input label. Put conversational labels or lengthy descriptions outside the Input box and use section headings. See  <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">Form design</a> guidelines for more information.
```

Comment:
In that first sentence, remove "the" after "Make" and make "label" plural. In the third sentence, replace "Put" with Place and add "of" after "outside." In the last sentence, "form design" should be lowercase. Also, not a big deal at all, but in the last sentence I'd swap word order to say, "For more information, see the form design guidelines.

Reply:
I've made copy changes based on your suggestions.

Reply:
Looks good. Leaving this comment thread unresolved though to see if Adam thinks my question about the image is worht worrying about. Thanks!

Reply:
@blackbaud-johnly Thanks for calling this out. Actually, using a 1-to-1 section heading per Input Box has its uses (low-density forms for consumer users, like a simple donation form) but should be avoided in others. That's part of what I want to dive into on the Form Design guidelines, but maybe it's also worth illustrating both approaches to labeling/explaining the inputs on a form here in this component.

Reply:
Sounds like we're OK with this screenshot as is, so resolving this conversation...